From: Jari Aalto <jari.aalto@cante.net>
Subject: Start GUI only when --gui option is used. Handle empty args.

---
 src/winetricks |   10 +++++++++-
 1 file changed, 9 insertions(+), 1 deletion(-)

--- a/src/winetricks
+++ b/src/winetricks
@@ -18548,6 +18548,14 @@
         shift
     fi
 
+    if [ ! "$1" ] && [ "$WINETRICKS_GUI" = "none" ] ; then
+       echo "Nothing to do. See --help. Graphical UI can be started with --gui"
+       exit 0
+    elif [ ! "$1" ] && [ "$WINETRICKS_GUI" != "none" ] ; then
+       # no arguments, but gui asked.
+       set -- gui
+    fi
+
     case "$1" in
     die) w_die "we who are about to die salute you." ;;
     volnameof=*)
@@ -18561,7 +18569,7 @@
         # This will read the volname from the given image and put it to stdout.
         winetricks_volname ${1#volnameof=}
         ;;
-    "")
+    gui)
         if test x"$DISPLAY" = x""
         then
             echo "DISPLAY not set, not defaulting to gui"
