From: Jari Aalto <jari.aalto@cante.net>
Subject: Define PATH for wineserver

---
 src/winetricks |   18 ++++++++++++++++++
 1 file changed, 18 insertions(+)

--- a/src/winetricks
+++ b/src/winetricks
@@ -3448,9 +3448,25 @@
         ;;
      *)
         WINE="${WINE:-wine}"
+
+	path=""
+
+	for tmp in /usr/lib/wine/wineserver \
+	           /usr/lib*/wine/wineserver \
+	           /usr/lib*/wine-unstable/wineserver
+	do
+	    if [ -x $tmp ]; then
+		path=$tmp
+		break;
+	    fi
+	done
+
         if test -x "${WINE}server"
         then
             WINESERVER="${WINE}server"
+        elif [ "$path" ]
+	then
+	    WINESERVER=$path					# Debian
         elif test -x "`dirname $WINE`/server/wineserver"
         then
             WINESERVER="`dirname $WINE`/server/wineserver"
@@ -3461,6 +3477,8 @@
             w_die "wineserver not found!"
         fi
 
+	unset tmp path
+
         if test "$WINEPREFIX"
         then
             WINETRICKS_ORIGINAL_WINEPREFIX="$WINEPREFIX"
