--- a/src/winetricks
+++ b/src/winetricks
@@ -4387,7 +4387,8 @@
     # 2014-09-30: 15.0  sha1sum 46341f1358bc6e0b9ddeae3591662a2ac68dc360
     # 2014-11-24: i15.0.0.356 sha1sum c0e6b8b1ed5ffa282945d21b21f8a5e03febc904
     # 2015-11-06: 19.x sha1sum 3bc2a568204a3a9b59ca347ab49585f0c5fab279
-    w_download http://airdownload.adobe.com/air/win/download/19.0/AdobeAIRInstaller.exe 3bc2a568204a3a9b59ca347ab49585f0c5fab279
+    # 2015-12-11: 19.x sha1sum ed0e55d3913f9eeae09c367fa5b796b6025945a4
+    w_download http://airdownload.adobe.com/air/win/download/19.0/AdobeAIRInstaller.exe ed0e55d3913f9eeae09c367fa5b796b6025945a4
     cd "$W_CACHE"/adobeair
     w_try "$WINE" AdobeAIRInstaller.exe $W_UNATTENDED_DASH_SILENT
 }
