Description: Drop shebang line from winetricks.bash-completion.
Author: Jens Reyer <jre.winesim@gmail.com>
Forwarded: https://github.com/Winetricks/winetricks/issues/1108

--- a/src/winetricks.bash-completion
+++ b/src/winetricks.bash-completion
@@ -1,5 +1,3 @@
-#!/bin/bash
-#
 # BASH completion for the Winetricks package manager
 #
 #--------------------------------------------------------------------
